lib/sync.js

```diff
@@ -13,7 +13,7 @@ module.exports = function (x, opts) {
     };
     var readFileSync = opts.readFileSync || fs.readFileSync;
     
-    var extensions = opts.extensions || [ '.js' ];
+    var extensions = opts.extensions || [ '.js', '.json' ];
```

Comment:
This is missing `.node` - instead, could we use the keys in `require.extensions`, which should ensure that the array always matches the require algorithm?

(if there's a concern that browserify won't be able to handle `require.extensions`, then let's add `.node` to the list also)